sdk/msg/msg.go

```diff
@@ -11,43 +11,43 @@ import (
 type Type int
 
 const (
-	TypeHello           Type = 100
-	TypeOpenRequest     Type = 200
-	TypeOpenResponse    Type = 201
-	TypePaymentRequest  Type = 300
-	TypePaymentResponse Type = 301
-	TypeCloseRequest    Type = 400
-	TypeCloseResponse   Type = 401
+	TypeHello           Type = 10
```

Comment:
❓ jw was this part of the gob addition? or something separate (im assuming bc the codes were too similar to http ones?)

Reply:
Gob stores integers in a variable int format. I think it is more complex than this, but it is something like 7 bits for the data bits and 1 bit to signal if there's more bytes. These integers are just identifiers for the different message types and they do not need to be that big. Changing them so they all fit inside one byte.